src/handlers/comment/handlers/allow.ts

```diff
@@ -20,13 +20,20 @@ export const setAccess = async (body: string) => {
     return;
   }
 
-  const regex = /^\/allow set-(\S+)\s@(\S+)\s(true|false)$/;
-
+  const regex = /\/allow\s+(\S+)\s+(\S+)\s+(\S+)/;
   const matches = body.match(regex);
 
   if (matches) {
-    const [, accessType, username, bool] = matches;
-
+    let accessType, username, bool;
+    matches.slice(1).forEach((part) => {
+      if (part.startsWith("@")) username = part.slice(1);
+      else if (part.startsWith("set-")) accessType = part.slice(4);
```

Comment:
why `.slice()`? isn't `part` already the full string like `set-multiplier`?
also you should check if variables are already set (not undefined) because somebody can put username twice like `/allow @example @example true`

Reply:
`accessType` should be one of ["priority", "time", "price", "multiplier"] not starting with set-
And do you mean `/allow @example @example true` is right?
In above case, there's no accessType and it will log error and will end...